grading/load_config_json.cpp

```diff
@@ -51,6 +51,12 @@ void AddDockerConfiguration(nlohmann::json &whole_config) {
   }
 
   bool docker_enabled = whole_config["docker_enabled"];
+
+  if (!whole_config["use_router"].is_boolean()){
+    whole_config["use_router"] = true;
```

Comment:
shouldn't be this be setting it false if use_router isn't a boolean/doesn't exist?